react-app/src/components/LemmaForm/LemmaForm.tsx

```diff
@@ -52,22 +75,27 @@ const LemmaForm = () => {
             </div>
             <div className="search formgroup">
                 <div className="text-toggle">
-                    <span className="text">Owner</span>
+                    <span className="text">Searchable</span>
                     <div className="toggle-con">
                     </div>
                 </div>
-                <div className="search-title">
-                    <label htmlFor="search-title">Title</label>
-                    <input type="search-title" name="title" id="search-title"/>
-                </div>
-                <div className="search-synopsis">
-                    <label htmlFor="search-synopsis">Synopsis</label>
-                    <input type="search-synopsis" name="synopsis" id="search-synopsis"/>
-                </div>
+                <Input 
+                name="search-title" changeHandle={changeSearchTitle}
+                placeholder="Search Title"
+                value={search.title}/>
+                <Input 
+                name="search-synopsis" changeHandle={changeSearchSynopsis}
+                placeholder="Search Synopsis"
+                value={search.synopsis}/>
             </div>
             <div className="recaptcha"></div>
             <input type="submit" value="Create Ref"/>
         </form>
+    <p>owner: {JSON.stringify(owner)}</p>
```

Comment:
Well done, Iyke. 

Do these here (owner, title etc.) need to be JSON stringified?

Reply:
It's for debugging purposes..... Not part of the form